lib/widget/camera.dart

```diff
@@ -358,12 +358,7 @@ class CameraScreenState extends WidgetState<CameraScreen>
           textbutton(
             title: 'Pick from gallery',
```

Comment:
@hamxa152 please don't hardcode these strings. This needs to come from resource file. I can't go over all your code looking for hardcoded strings. Please remove all hardcoded strings and make sure they come from resource files.